### PR TITLE
feat(icons): add TwiML Bins product icon

### PR DESCRIPTION
**Pull request by kateschwarz**
Add TwiML Bins product icon

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by serifluous** [comment]
Hey @kateschwarz. Can you send me the SVG for this icon so I can add it to our Sketch library? I might be able to compress it a bit more, too.

**Comment by kateschwarz** [comment]
@serifluous I removed the original SVG and replaced it with the compressed version you provided. Thanks!

**Review by serifluous — APPROVED** [review]

**Review by TheSisb — APPROVED** [review]

File: packages/paste-icons/src/ProductTwimlBinsIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (TheSisb) [inline packages/paste-icons/src/ProductTwimlBinsIcon.tsx:1]: This file name has incorrect capitalization. Can you rename it to `ProductTwiMLBinsIcon.tsx` please?

Renaming files in git on osx is awkward, you might need this command: `git mv ProductTwimlBinsIconProps.tsx ProductTwiMLBinsIconProps.tsx` for git to recognize the new filename

**Review by TheSisb — CHANGES_REQUESTED** [review]
This rename should fix breaking builds.

File: packages/paste-icons/src/ProductTwimlBinsIcon.tsx (hunk 1 above)
Comment (kateschwarz) [reply packages/paste-icons/src/ProductTwimlBinsIcon.tsx:1]: Super weird... it was the correct name in my text editor and when I would `ls` all of the files in that folder, but I ran the command you suggested above, and looks like it changed the name 👍

Comment (TheSisb) [reply packages/paste-icons/src/ProductTwimlBinsIcon.tsx]: Yeah it's an issue with macs. You have to do the `git mv` command if you're renaming a file that is already checked in by just changing capitalization. Thanks!

**Review by TheSisb — APPROVED** [review]

**Pull request merged**